.github/scripts/backfill-zenodo-authors.py

```
@@ -0,0 +1,211 @@
+        except Exception as exc:  # noqa: BLE001 — one bad record shouldn't abort the batch
+            failed += 1
+            # Best-effort: discard any half-open draft so a broken edit never shadows
+            # the live published record.
+            try:
+                api_request(
+                    f"{args.api_url}/api/deposit/depositions/{record_id}/actions/discard",
+                    method="POST", token=args.zenodo_token, retries=1,
+                )
+                print(f"        rolled back draft for record {record_id}")
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)